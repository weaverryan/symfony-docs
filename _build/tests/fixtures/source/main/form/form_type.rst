FormType Documentation
======================

.. _internal-reference:
