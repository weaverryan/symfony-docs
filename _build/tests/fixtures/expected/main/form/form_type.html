<!DOCTYPE html>
<html>
<head>
    <meta charset="utf-8" />
</head>
<body>
<h1 id="formtype-documentation">FormType Documentation<a class="headerlink" href="#formtype-documentation" title="Permalink to this headline">Â¶</a></h1>
<span id="internal-reference"></span>

</body>
</html>
